pkg/controller/gitopsconfig/controller.go

```diff
@@ -51,6 +52,13 @@ const kubeGitopsFinalizer string = "eunomia-finalizer"
 // PushEvents channel on which we get the github webhook push events
 var PushEvents = make(chan event.GenericEvent)
 
+//CommitID stores the commitid of template source
+var CommitID string
+
+func init() {
+	CommitID = os.Getenv("TEMPLATE_SHA1")
```

Comment:
Did you test that this works? This Go file is part of eunomia operator binary, which AFAIK is normally run in a different Pod than template processors.  I believed that the `template-processors/base/bin/gitClone.sh` script where you seem to be setting TEMPLATE_SHA1 is run as part of template processors, is it not so? If the line above gets non-empty TEMPLATE_SHA1, can you explain to me how? I guess I must be missing something?